commons/src/test/groovy/org.codice.ddf.admin.commons.requests/RequestUtilsTest.groovy

```diff
@@ -106,32 +113,38 @@ class RequestUtilsTest extends Specification {
         !report.containsFailureMessages()
         report.messages().find {it.type() == WARNING && it.subtype() == RequestUtils.UNTRUSTED_CA}
         report.messages().find {it.type() == SUCCESS && it.subtype() == RequestUtils.EXECUTED_REQUEST}
+
+        2 * utils.closeClientAndResponse(_, _)
```

Comment:
It should close once for each try, the first with the standard trust policy, and the second with the trustAnyCA policy. I'd say this is less testing and implementation detail and more a test that we're doing what we're expecting. If we're not opening and closing 2 clients, something is off.